### PR TITLE
Support scoped modules.

### PR DESCRIPTION
**Pull request by hemanth**
^

### CODE REVIEW

**Comment by bahmutov** [comment]
+1 on this one

**Comment by bahmutov** [comment]
I switched to using @hemanth fork in my https://github.com/bahmutov/grunt-nice-package for now

File: test/qunit-pjv.js [hunk 1]
```diff
@@ -62,7 +62,8 @@ QUnit.test("Dependencies Ranges", function() {
             'tilde-first': '~1.2',
             'x-version': '1.2.x',
             'tilde-top': '~1',
-            'caret-top': '^1'
+            'caret-top': '^1',
+            "@es-shims/api": "^1.0.0"
```
Comment (LinusU) [inline test/qunit-pjv.js:66]: It would have been nice to follow the existing style of single quotes here...

**Comment by LinusU** [comment]
ping @gorillamania would really appreciate getting this merged. I'd be happy to step up as a maintainer if you no longer have time to devote to this project :)

**Comment by TechNickAI** [comment]
Yes please! Note this has conflicts that prevent it from being merged. What is your npm username?I'll add you as a maintainer

**Comment by LinusU** [comment]
Glad to hear! I'm [linusu](https://npmjs.com/~linusu) on npm :)

**Comment by TechNickAI** [comment]
@LinusU I've added you as a collaborator on this repo and on NPM. Really appreciate the help!

**Comment by hemanth** [comment]
Meow.

File: test/qunit-pjv.js (hunk 1 above)
Comment (JoshuaKGoldberg) [reply test/qunit-pjv.js:66]:
```suggestion
            '@es-shims/api': '^1.0.0'
```

**Review by JoshuaKGoldberg — APPROVED** [review]
🚀 Looks great! Just the one style point on keeping to the existing `'` for the added line.

**Comment by JoshuaKGoldberg** [comment]
@hemanth I don't have permissions to push the suggestion (or merge conflict resolutions) to your branch. Would you be up for either doing that yourself or giving me permission to?

I know it's been a long time, but I'd love to get this in 😄

**Comment by hemanth** [comment]
@JoshuaKGoldberg I guess @bahmutov has a better fix :)

**Comment by JoshuaKGoldberg** [comment]
Sorry, I don't follow - what's the better fix?

**Comment by hemanth** [comment]
https://github.com/TechNickAI/package.json-validator/pull/48#issuecomment-161726665 

Looks like my fork is deleted, so ^ :|

**Comment by JoshuaKGoldberg** [comment]
Ah, gotcha.

For context: I want to accept these changes and give you credit as the author. Do you want to re-fork and send a new PR?

If not / you don't have time, no worries, I can always make them myself and list you as a co-author. But I'd feel bad doing that without giving you the opportunity to be the primary author. 🙂

**Comment by hemanth** [comment]
@JoshuaKGoldberg That's ok, FOSS FTW! ❤️

**Pull request closed without merging**